tools/OloHeaderTool/main.cpp

```
@@ -0,0 +1,842 @@
+                    }
+
+                    // Custom getter/setter — full expressions using 'comp'
+                    if (auto it = meta.find("Get"); it != meta.end())
+                        prop.customGet = it->second;
+                    if (auto it = meta.find("Set"); it != meta.end())
+                        prop.customSet = it->second;
+
+                    // Find or create ComponentDef
+                    if (compMap.find(currentComponent) == compMap.end())
+                    {
+                        compMap[currentComponent] = components.size();
```

Cppcheck (2.20.0)

[performance] Searching before insertion is not necessary. Instead of 'compMap[currentComponent]=components.size()' consider using 'compMap.try_emplace(currentComponent, components.size());'.

(stlFindInsert)